_posts/we-need-insights-not-data.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: We need insights, not data
+author: Mauro Servienti
+title: "Gauges and graphs attract software engineers like honey for bees. We spend hours implementing distributed logging solutions or monitoring systems, and still we have a hard time understanding what's going on."
+header_image: /img/posts/we-need-insights-not-data/header.jpg
+tags:
+- personal
+- distributed-systems
+---
+
+I used a [Whoop](https://www.whoop.com) device for about 18 months. What's Whoop, you might wonder?
+
+Quoting from their website:
+
+> The latest, most advanced fitness and health wearable available. Monitor your recovery, sleep, training, and health, with personalized recommendations and coaching feedback.
+
+I stopped using it because I got no personalized recommendations and coaching feedback. And if you recall, it's the thing I talked about at the [end of 2021 in "Do I still need a smartwatch?"](https://milestone.topics.it/2021/12/27/do-I-still-need-a-smartwatch.html):

```

Comment:
I suggest removing references to readers who follow regularly. The post may get more hits from a Google search or some other random visitor who may not recall this at all.

```suggestion
I stopped using it because I got no personalized recommendations and coaching feedback. I mentioned this at the [end of 2021 in "Do I still need a smartwatch?"](https://milestone.topics.it/2021/12/27/do-I-still-need-a-smartwatch.html):

```